### ISSUE TITLE
[BUG] <能否基于chat模型使用model.generate函数>

### ISSUE THREAD

**Issue by dengfenglai321** [QwenLM/Qwen-VL#72]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

能否基于chat模型使用model.generate函数？
还是chat模型必须要使用model.chat（）

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by ShuaiBai623** [comment]
chat模型可以使用model.generate函数，可以参考model.chat进行格式规范

**Comment by Keep-lucky** [comment]
> chat模型可以使用model.generate函数，可以参考model.chat进行格式规范

请问格式规范应该如何使用，因为我尝试了model.generate函数和model.chat函数，前者效果远不如后者，请问使用model.generate函数应该如何进行格式规范呢？

**Comment by wangrunchuan** [comment]
> > chat模型可以使用model.generate函数，可以参考model.chat进行格式规范
> 
> 请问格式规范应该如何使用，因为我尝试了model.generate函数和model.chat函数，前者效果远不如后者，请问使用model.generate函数应该如何进行格式规范呢？

有一个点是在使用 generate 函数时数据要 left padding

**Comment by yihp** [comment]
@wangrunchuan@Keep-lucky你好老师，请问你们最后怎么更改的？我按照eval_mm
/evaluate_caption.py这个评估脚本里的generate 函数进行评估，效果很差，基本就是乱回答，但是用chat函数就好很多

**Comment by yihp** [comment]
@wangrunchuan @ShuaiBai623 @dengfenglai321 @Keep-lucky 你好老师，请问你们最后怎么更改的？我按照eval_mm
/evaluate_caption.py这个评估脚本里的generate 函数进行评估，效果很差，基本就是乱回答，但是用chat函数就好很多
